### ISSUE TITLE
Could someone publish the latest version(10.25.0) on Firefox?

### ISSUE THREAD

**Issue by Holybasil** [MetaMask/metamask-extension#17889]
### Describe the bug

"wallet_addEthereumChain" will lead "May not specify default MetaMask chain" which works well in Chrome

### Steps to reproduce

```js
wallet_addEthereumChain({
  chainId: "0x5",
  chainName: "Goerli",
  rpcUrls: ["https://endpoints.omniatech.io/v1/eth/goerli/public"],
    ...
})
```

### Version

10.22.2

### Build type

None

### Browser

Firefox

### Operating system

MacOS

**Comment by kremalicious** [comment]
duplicate of #17604

**Comment by FrederikBolding** [comment]
Version `10.25.0` has been submitted for review with Firefox on the same date we submitted to Chrome. The review process on Firefox sometimes requires a manual review by a Firefox engineer and for this specific release they were unable to reproduce the build we submitted. We have submitted new and improved build instructions to Firefox and are waiting for them to re-review the submitted version.
